Checkout/Api/Platform/Cart.php

```diff
@@ -126,6 +126,40 @@ public static function estimateShippingMethods(
         }
     }
 
+    /**
+     * Gets the quote total information.
+     *
+     * @param Mage_Core_Controller_Request_Http $request
+     * @param Mage_Core_Controller_Response_Http $response
+     * @return Mage_Core_Controller_Response_Http
+     */
+    public static function totalsInformation(
```

Comment:
This is so similar to the previous two functions. Is there a reason we can't refactor these to have a little less repetition?

Reply:
As there are no tests around this I'd rather not refactor other functions to avoid potential issues. This function may also be updated in the future to implement more of the M2 spec for the endpoint of the same name which would deviate from the similarity of the other functions